NitroxClient/MonoBehaviours/Gui/MainMenu/JoinServer.cs

```diff
@@ -157,7 +157,17 @@ private void StartMultiplayerClient()
             }
             catch (ClientConnectionFailedException)
             {
-                Log.InGame($"Unable to contact the remote server at: {ServerIp}:11000");
+                string Splitter = ":";
```

Comment:
Instead of using an if check on the ipaddress string I would just always include the port number given. You can use `multiplayerSession.Port` if you added it.

`Log.InGame($"Unable to contact the remote server at: {multiplayerSession.IpAddress}:{multiplayerSession.Port}");`